ngraph/src/ngraph/op/proposal.cpp

```diff
@@ -77,20 +75,22 @@ void op::Proposal::validate_and_infer_types()
 
         auto batch_size = class_probs_shape[0];
         set_output_type(0, get_input_element_type(0), Shape{batch_size * m_attrs.post_nms_topn, 5});
+        set_output_type(1, get_input_element_type(0), Shape{batch_size * m_attrs.post_nms_topn});
```

Comment:
Why do you add additional output for v0 Proposal? I thought that you use Proposal v4 for this case.

Reply:
removed.